recipes/cdsdashboards/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "cdsdashboards" %}
+{% set version = "0.0.19" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ccc6eebb9e5471e6dcef8bc7d028561a8c8649b5d7b8dfcb3472ba3796fe7867
+
+build:
+  noarch: python
+  number: {{ build_number }}
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      number: {{ build_number }}
+      noarch: python
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
```

Comment:
If you rename these to `build.sh` and `bld.bat`, these two line are not needed, and therefore you don't need any selectors

Reply:
If I rename the script, it does not work as it tries to use them for the fake _meta_ package that comes with no host nor build environment.

Reply:
Ahhh OK  - just keep the build_base.sh one then.  It is OK if this fails on windows/  Noarch packages are only built on linux

Reply:
Thanks for the tip. I made the correction.